test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -98,12 +98,6 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           `window.ethereum.request(${switchEthereumChainRequest})`,
         );
 
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-        await driver.findElement({
-          text: 'Use your enabled networks',
-          tag: 'p',
-        });
-
```

Comment:
this check should still be valid. The globally selected network is 0x53a when Dapp Two connects, meaning that 0x539 should not be connected for Dapp Two and a Chain Permission approval should appear when trying to switch to it

Reply:
Yes, At this point 0x539 is not connected for both Dapp Dapp two. But the comment here is wrong since we are initiating this request from Dapp One and since Dapp One is connected to 0x539, it will never render this screen. 
![Screenshot 2024-10-25 at 7 24 22 PM](https://github.com/user-attachments/assets/136f8470-e343-4661-aec5-0899d0bd5879)

Updated this here and Dapp One is not connected to 0x539 while making initial connection https://github.com/MetaMask/metamask-extension/pull/27980/commits/7f7646f7a81ee1c998cfddde8a2185b9099fa80c. Also, updated comments here to avoid confusion https://github.com/MetaMask/metamask-extension/pull/27980/commits/4f6bad59a8e658192840ec4942e6dab7ae42e62e (could have been in the same commit)